src/types/venues.ts

```diff
@@ -160,7 +160,6 @@ export interface BaseVenue {
     text: string;
   };
   showLearnMoreLink?: boolean;
-  liveScheduleOtherVenues?: string[];
```

Comment:
It's cool that you caught this. How did you do it though? Did you search for `liveSchedule` in this file and just noticed the extra field?

Reply:
Haha yup! And then my IDE has a ‘find references’ thing that told me it was unused. Then I did a find across all files for the name to confirm and also couldn’t find usages.